src/git-istage/git-istage.csproj

```diff
@@ -15,12 +15,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LibGit2Sharp" Version="0.26.0-preview-0070" />
+    <PackageReference Include="LibGit2Sharp" Version="0.26.2" />
+
```

Comment:
Why the blank line?

Reply:
For readability, separate one-line ref from one with children tags.

Reply:
Fair enough :-)